README.md

```diff
@@ -1 +1,37 @@
-# hasek
\ No newline at end of file
+# hasek
```

Comment:
needs more pictures of Hasek [doing his thing](https://i.ytimg.com/vi/YBA_k1NSgnI/maxresdefault.jpg)